src/main/java/models/Author.java

```diff
@@ -4,7 +4,14 @@
  * Created by Szilvi on 2017. 10. 04..
  */
 public class Author {
-
+    /**
+     * Itt igazából az egész getter setter dolog ködös nekem. Ahhoz hogy ez valid legyen ahhoz ugye private -ra kell
+     * a name és date properyket állítani. Viszont mivel nincs semmi állítgatás a set és a get metódusnál se ezért sztem ezeket elhagyhatod
+     * és csak simán public String name -ként venném fel.
+     * A másik dolog ez már csak szép érzek, nem tudom hogy van pontosan a Javaban hogy először az összes setter metódust felsorold vagy
+     * inkább propertynként csinálod a setet és a getet (én a c# miatt ezt preferálom) de ezt próbáld egy féleképpen csinálni.
+     * Láttam hogy az egyik osztályod így van, a másik úgy.
+     */
```

Comment:
Én am személy szerint azt tartom a legtisztábbnak ha ezek az access modifierek ki vannak írva. Tudsz példát mutatni hogy mikor használod a gettert settert?

Reply:
A GithubService classomban mindenhol, és az alábbi miatt hasznosak:
When you need to change the behavior or regulate what you can set on a variable. For instance, lets say you had a setSpeed(int speed) method. But you want that you can only set a maximum speed of 100. You would do something like:

public void setSpeed(int speed) {
  if ( speed > 100 ) {
    this.speed = 100;
  } else {
    this.speed = speed;
  }
}
Now what if EVERYWHERE in your code you were using the public field and then you realized you need the above requirement? Have fun hunting down every usage of the public field instead of just modifying your setter.